### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent error with static short closure

### ISSUE THREAD

**Issue by sspat** [squizlabs/PHP_CodeSniffer#2570]
# Versions
codesniffer@dev-master
php 7.4.0beta1

# Description
When adding static to a short syntax closure which is an argument of a multi-line function call an incorrect indent error is raised.

```php
<?php

declare(strict_types=1);

$numbers = [1, 2, 3];

$result = array_map(
    static fn(int $number) : int => $number + 1,
    $numbers
);
```
```
 8 | [ERROR | [x] [Line indented incorrectly; expected 0 spaces, found 4 (Generic.WhiteSpace.ScopeIndent.IncorrectExact)
```

# Expected behavior
The same code with a non-static short closure raises no errors, the same should apply to a static one.
```php
<?php

declare(strict_types=1);

$numbers = [1, 2, 3];

$result = array_map(
    fn(int $number) : int => $number + 1,
    $numbers
);
```

**Comment by jrfnl** [comment]
Arrow functions are not supported yet. Please see: #2523

**Comment by sspat** [comment]
@jrfnl Sorry, missed that one. What is the best way i can help with the issues i find, and is it helpful at all?

**Comment by jrfnl** [comment]
@sspat For now, not much can be done until a decision is taken about #2523 and the `T_FN` token is backfilled.

Once that's done, it'll be useful to know which sniffs need looking at. I'd suggest adding code samples like the one above and the other issues reported to the thread in #2523 to keep things together (or at the very least to link the issues by mentioning the canonical issue number).

For anything missed in the initial accounting for arrow functions, so things found _after #2523 has been addressed and closed_, new issues would seem appropriate.

**Comment by gsherwood** [comment]
@sspat As @jrfnl, the best thing to do is put comments on the main issue so it can all be addressed at once. I'll merge the 3 issues you submitted with the main one, but can you please add more comments to #2523 if you come across more examples. Thanks.

**Comment by gsherwood** [comment]
Closing as merged with #2523
